data/assets/examples/scale/luascale/example.lua

```diff
@@ -0,0 +1,3 @@
+function scale(simTime, prevSimTime, wallTime)
```

Comment:
Is it possible to have this function as part of the asset? It looks like it only accepts a path but I thought I'd ask anyways. I'm thinking it won't show up in the documentation (unless we make some changes)